internal/servers/controller/handlers/credentiallibraries/credentiallibrary_service.go

```diff
@@ -675,3 +714,48 @@ func validateMapping(badFields map[string]string, credentialType string, overrid
 		}
 	}
 }
+
+func getMappingUpdates(credentialType credential.Type, current vault.MappingOverride, new map[string]interface{}, apiMasks []string) (map[string]interface{}, bool) {
+	ret := make(map[string]interface{})
+	masks := make(map[string]bool)
+	for _, m := range apiMasks {
+		if m == credentialMappingPathField {
+			// got top level credential mapping change request, this mask
+			// can only be provided when clearing the entire override.
+			return nil, true
+		}
+
+		if s := strings.SplitN(m, "credential_mapping_overrides.", 2); len(s) == 2 {
```

Comment:
this should probably use `credentialMappingPathField` instead of hard coding `credential_mapping_overrides` again.

Reply:
ty